Find a functionally equivalent code:android.content.pm.PackageItemInfo.loadIcon:COMMENT
Method Modifier: public      
Comment:/**
 * Retrieve the current graphical icon associated with this item.  This
 * will call back on the given PackageManager to load the icon from
 * the application.
 *
 * @param pm A PackageManager from which the icon can be loaded; usually
 * the PackageManager from which you originally retrieved this item.
 *
 * @return Returns a Drawable containing the item's icon.  If the
 * item does not have an icon, the item's default icon is returned
 * such as the default activity icon.
 */

@@ -1,9 +1,3 @@
 {
-    if (icon != 0) {
-        Drawable dr = pm.getDrawable(packageName, icon, getApplicationInfo());
-        if (dr != null) {
-            return dr;
-        }
-    }
-    return loadDefaultIcon(pm);
+    return pm.loadItemIcon(this, getApplicationInfo());
 }

